build.gradle

```diff
@@ -184,18 +179,11 @@ project(':api') {
         }
     }
 
-    artifacts {
-        archives javadocJar, sourcesJar, shadowJar
-    }
-
```

Comment:
This is unnecessary as far as I can see.

Reply:
Actually this is needed @marcphilipp